webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue

```diff
@@ -161,9 +161,8 @@
         </a>
       </div>
 
-      <v-card-text 
-        class="spaceCardBody align-center"
-        :class="!isMobile && 'pt-2 pb-1'">
+      <v-card-text
+        class="spaceCardBody align-center py-auto py-sm-2">
```

Comment:
`py-auto` doesn't exit as class do you mean `py-0` ?

```suggestion
        class="spaceCardBody align-center py-0 py-sm-2">
```

Reply:
Nope, it is not py-0, it is not needed for the mobile
I just applied py-sm-2